.buildkite/pipeline.yml

```diff
@@ -0,0 +1,9 @@
+steps:
+  - label: ':github: Run submodule build'
+    branches: master
+    agents:
+      Environment: production
```

Comment:
is upgrade37 going to run on production environment? are the PHP versions not different?